microshift-manifests/0000_50_olm_00-pprof-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    release.openshift.io/create-only: "true"
```

Comment:
This is an example of 1 file where the regular manifest has `capability.openshift.io/name: "OperatorLifecycleManager"` but the MicroShift variant does not. I think this capability annotation should be on the MicroShift manifests too?

Reply:
done, PTAL